frontend/src/lib/components/HistoryList/HistoryList.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react'
+import { ProfilePicture } from 'lib/components/ProfilePicture'
+import './HistoryList.scss'
+import { LemonTable, LemonTableColumns } from 'lib/components/LemonTable'
+import { TZLabel } from 'lib/components/TimezoneAware'
+import { historyListLogic, HumanizedHistoryListItem } from 'lib/components/HistoryList/historyListLogic'
+import { useActions, useValues } from 'kea'
+
+interface HistoryListProps {
+    type: 'FeatureFlag'
+    id: number
+}
+
+export const HistoryList = ({ type, id }: HistoryListProps): JSX.Element => {
+    const logic = historyListLogic({ type })
+    const { history, historyLoading } = useValues(logic)
+    const { fetchHistory } = useActions(logic)
+    useEffect(() => {
+        fetchHistory(id)
+    }, [type, id])
```

Comment:
This `useEffect` feels out of place here. I'm still not sure I fully understand why the logic has to be keyed by type, but not by id. Won't block for it, but 🤔